projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.service.ts

```diff
@@ -8,6 +13,8 @@ export class OrderDetailsService {
   orderCode$: Observable<string>;
   orderLoad$: Observable<{}>;
 
+  _cancellationReturnRequestInputs: CancellationReturnRequestEntryInput[];
```

Comment:
can't we make this private. It's still possible to do `orderDetails.__cancellationReturnRequestInputs = ...`

Reply:
yes, it's missing